### ISSUE TITLE
Text in Hero Widget in not scaling as it should on my Mobile

### ISSUE THREAD

**Issue by brendensharp** [godaddy/wp-stout-theme#33]
I'm using your Stout theme which I think is FANTASTIC however the Header Text isn't scaling on my mobile (Galaxy S7 Edge)

What do I have to do to make the Header Text (Hero Widget) scale responsively so that it fits on my mobile?

At the moment it's cutting the word in half and the remaining letters appear on the next line.

AUTOM
ATIC
SPORT
S
PRODU
CTION
SYSTE
M

It's meant to read

On my PC it reads

AUTOMATIC
SPORTS
PRODUCTION
SYSTEM

www.automatictv.com.au

Thanks in advance

Brenden Sharp

**Comment by EvanHerman** [comment]
Hi Brenden,

I just took a look on my mobile device (iPhone 5), and it looks like it is rendering properly on my end. Did you alter some CSS by chance?

![Example](https://cldup.com/0EEUo0hp7Z.png)

I did setup a test instance with the same text you have on your site, and it looks like I am hitting that error as well. This is something that we are going to need to patch up and get included in the next release of stout.

Evan
